ci/kind/kind-setup.sh

```diff
@@ -242,6 +244,32 @@ function configure_extra_networks {
   done
 }
 
+function configure_kind_ipam_vlan_subnets {
+  if [[ -z $IPAM_VLAN_SUBNETS || -z $IPAM_VLAN_IDS ]]; then
+    return
+  fi
+  
```

Comment:
if subnets and vlans are provided for ipam tests , must echo message like "configuring flexible ipam kind networks"